src/main/java/mate/academy/spring/boot/controller/BookController.java

```diff
@@ -27,29 +31,43 @@ public BookController(BookService bookService) {
         this.bookService = bookService;
     }
 
-    @GetMapping
-    public List<BookDto> getAll() {
-        return bookService.getAll();
-    }
-
-    @ResponseStatus(HttpStatus.NO_CONTENT)
     @DeleteMapping("/{id}")
-    public void delete(@PathVariable Long id) {
+    @ResponseStatus(HttpStatus.NO_CONTENT)
+    @Operation(summary = "Delete the book", description = "Delete the book by ID")
+    public void deleteBookById(@PathVariable Long id) {
         bookService.deleteById(id);
     }
 
     @GetMapping("/{id}")
+    @Operation(summary = "Get books", description = "Get the book by ID")
     public BookDto getBookById(@PathVariable Long id) {
```

Comment:
```suggestion
    public BookDto getBookById(@PathVariable @Positive Long id) {
```
Don't forget to add `@Validated` under the controller